docs/guide/browser.md

```diff
@@ -6,6 +6,56 @@ title: Browser Mode | Guide
 
 This page provides information about the experimental browser mode feature in the Vitest API, which allows you to run your tests in the browser natively, providing access to browser globals like window and document. This feature is currently under development, and APIs may change in the future.
 
+## Installation
+
+::: code-group
```

Comment:
Maybe it's better to have several sections - "With playwright provider", "With webdriverio provider" and "With `none` provider", otherwise people will just install everything without reading

Reply:
Good idea, I'll do that. I had a thought that the comment isn't good enough.